docs/topics/request-response.rst

```diff
@@ -172,6 +172,18 @@ Request objects
        is given in the ``meta`` argument). See also
        :ref:`topics-request-response-ref-request-callback-arguments`.
 
+    .. method:: Request.urljoin(uri)
```

Comment:
Stdlib's urljoin calls argument "url", not "uri", and I think it is more correct.